test/gurka/gurka.cc

```diff
@@ -695,7 +698,48 @@ valhalla::Api do_action(const valhalla::Options::Action& action,
   if (!request_json) {
     request_json = &dummy_request_json;
   }
-  *request_json = detail::build_valhalla_request(location_type, lls, costing, options, stop_type);
+  *request_json = detail::build_valhalla_request({location_type}, {lls}, costing, options, stop_type);
+  return do_action(action, map, *request_json, reader, response);
+}
+
+// overload for /sources_to_targets
+valhalla::Api do_action(const valhalla::Options::Action& action,
+                        const map& map,
+                        const std::vector<std::string>& sources,
+                        const std::vector<std::string>& targets,
+                        const std::string& costing,
+                        const std::unordered_map<std::string, std::string>& options,
+                        std::shared_ptr<valhalla::baldr::GraphReader> reader,
+                        std::string* response,
+                        std::string* request_json) {
```

Comment:
another overload, unfortunately. I could've changed the signature of the `do_action` but it's public so I'd have to change every gurka test almost.